boule/_ellipsoid.py

```diff
@@ -394,6 +404,25 @@ def gravity_pole(self):
         )
         return result
 
+    def __str__(self):
+        s = self.name + " - " + self.long_name + "\n"
+        s += "Oblate ellipsoid:\n"
+        s += f"  Semimajor axis: {self.semimajor_axis} m\n"
+        s += f"  Flattening: {self.flattening}\n"
+        s += f"  GM: {self.geocentric_grav_const} m³/s²\n"
+        s += f"  Angular velocity: {self.angular_velocity} rad/s"
```

Comment:
Just a small suggestion to implement an output similar to what @leouieda proposed in #181. What do you think?

```suggestion
        n_spaces = 16
        s = self.name + " - " + self.long_name + "\n"
        s += "Oblate ellipsoid:\n"
        s += f"  • {'Semimajor axis':>{n_spaces}}: {self.semimajor_axis} m\n"
        s += f"  • {'Flattening':>{n_spaces}}: {self.flattening}\n"
        s += f"  • {'GM':>{n_spaces}}: {self.geocentric_grav_const} m³/s²\n"
        s += f"  • {'Angular velocity':>{n_spaces}}: {self.angular_velocity} rad/s"
```